Tools/gsc/config.yaml.template

```diff
@@ -0,0 +1,7 @@
+Distro: "ubuntu18.04"
+Graphene:
+    Repository: "https://github.com/oscarlab/graphene.git"
+    Branch: "master"
+SGXDriver:
+    Repository: "https://github.com/01org/linux-sgx-driver.git"
+    Branch: "sgx_driver_1.9"
```

Comment:
Any reason/limitation for not using latest SGX driver tag `sgx_driver_2.6` ? I tried with `sgx_driver_2.6` and GSC works fine for me in Azure ACC VM.